### DISCUSSION TITLE
display camera stream along with floorplan

### DISCUSSION THREAD

**Discussion by jtcookhome** [ExperienceLovelace/ha-floorplan#437] [Q&A]
I want to show my 4 camera streams to the right of my floorplan where I have the white box.  Can this be done?  Is there an example I can follow?
![image](https://github.com/ExperienceLovelace/ha-floorplan/assets/140983404/1feeb0a7-deca-447e-84eb-fceb2f25b288)

**Comment by OzGav** [comment]
Try this https://www.home-assistant.io/dashboards/sidebar/

**Comment by Trzinka** [comment]
You can see my settings 
https://github.com/ExperienceLovelace/ha-floorplan/discussions/411

**Comment by exetico** [comment]
It depends on how your camera's are exposed. Take a look at our docs:

https://experiencelovelace.github.io/ha-floorplan/docs/example-home/

<img width="770" alt="image" src="https://github.com/ExperienceLovelace/ha-floorplan/assets/3549445/f4cc6f0e-a1dd-44b4-ad9f-913f42256413">


There's an example on how to do something like what you're trying to do. With that said, I'd personally use a sidebar, as OzGav suggested.

Please follow up by marking your preferred suggestion as a answer. Or provide your own solution, if you managed to find something even more cool. 😄
